h2o-algos/src/main/java/hex/tree/gbm/GBM.java

```diff
@@ -96,9 +93,11 @@
         error("_offset_column", "Offset is not supported for "+_parms._distribution+" distribution.");
       }
       if (_parms._monotone_constraints != null && _parms._monotone_constraints.length > 0 &&
-              !(DistributionFamily.gaussian.equals(_parms._distribution) || 
-                      DistributionFamily.bernoulli.equals(_parms._distribution) || DistributionFamily.tweedie.equals(_parms._distribution))) {
-        error("_monotone_constraints", "Monotone constraints are only supported for Gaussian and Bernoulli distributions, your distribution: " + _parms._distribution + ".");
+              !(DistributionFamily.gaussian.equals(_parms._distribution) ||
+                      DistributionFamily.bernoulli.equals(_parms._distribution) ||
```

Comment:
creating a method (like !supportsMonotoneConstraints(distribution)) or a list of distributions supporting monotone constraints would make the condition more readable